io.github.DonaldKellett.csnaketerm.yml

```diff
@@ -0,0 +1,41 @@
+app-id: io.github.DonaldKellett.csnaketerm
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: run.sh
+modules:
+  - 'shared-modules/python2.7/python-2.7.json'
+  - name: nodejs
+    buildsystem: simple
+    build-commands:
+      - ./configure --prefix=/app
+      - make -j$FLATPAK_BUILDER_N_JOBS
+      - make install
+    sources:
+      - type: archive
+        url: https://nodejs.org/dist/latest-v10.x/node-v10.23.1.tar.xz
+        sha256: 88aa16f5af79615b183ca55ed81393763169e75d1fb96013cf1831895c6cedfa
+  - name: csnaketerm
+    buildsystem: simple
+    build-commands:
+      - make prefix=/app install
+    sources:
+      - type: archive
+        url: https://github.com/DonaldKellett/csnaketerm/archive/v0.2.0.tar.gz
+        sha256: 96b40e564e90c4b7395001d0149e45f2bce57d505da79424e19d7ff38441fa79
+  - name: metainfo
+    buildsystem: simple
+    build-commands:
+      - install -D io.github.DonaldKellett.csnaketerm.metainfo.xml /app/share/metainfo/io.github.DonaldKellett.csnaketerm.metainfo.xml
+    sources:
+      - type: file
+        path: io.github.DonaldKellett.csnaketerm.metainfo.xml
+  - name: run.sh
+    buildsystem: simple
+    build-commands:
+      - install -D run.sh /app/bin/run.sh
+    sources:
+      - type: file
+        path: run.sh
```

Comment:
There's no need to make these separate modules, you can merge it with `csnaketerm`.